clients/src/main/java/org/apache/kafka/clients/admin/internals/AlterConsumerGroupOffsetsHandler.java

```diff
@@ -179,6 +179,9 @@ private void handleError(
             case INVALID_GROUP_ID:
             case INVALID_COMMIT_OFFSET_SIZE:
             case GROUP_AUTHORIZATION_FAILED:
+            // Member level errors.
+            case UNKNOWN_MEMBER_ID:
+            case FENCED_INSTANCE_ID:
```

Comment:
When would we encounter this error? My understanding is that it'll only be returned when the broker receives a request with a group instance ID defined, and IIUC it's not possible to define one with the admin API we expose right now.

Reply:
You're right, currently this error code is not expected as the admin client does not set the group instance ID. I've removed it.